core/src/cluster_slots_service.rs

```diff
@@ -100,12 +100,24 @@ impl ClusterSlotsService {
             lowest_slot_elapsed.stop();
             let mut process_cluster_slots_updates_elapsed =
                 Measure::start("process_cluster_slots_updates_elapsed");
-            if let Some(slots) = slots {
-                Self::process_cluster_slots_updates(
-                    slots,
-                    &cluster_slots_update_receiver,
-                    &cluster_info,
-                );
+
+            let node_id = cluster_info.id();
+            let my_stake = bank_forks
+                .read()
+                .unwrap()
+                .root_bank()
+                .current_epoch_stakes()
+                .node_id_to_stake(&node_id)
+                .unwrap_or_default();
```

Comment:
This probably should use `EpochSpecs`:
https://github.com/anza-xyz/agave/blob/e12e6fbf7/gossip/src/epoch_specs.rs#L18

Reply:
Done in [e3df68e](https://github.com/anza-xyz/agave/pull/5141/commits/e3df68e1916f208e86fc1c1709a697a5518fe0ff)